packages/@n8n/nodes-langchain/package.json

```diff
@@ -137,13 +138,15 @@
     "@getzep/zep-cloud": "1.0.6",
     "@getzep/zep-js": "0.9.0",
     "@google-ai/generativelanguage": "2.5.0",
+    "@google-cloud/resource-manager": "^5.3.0",
     "@google/generative-ai": "0.11.4",
     "@huggingface/inference": "2.7.0",
     "@langchain/anthropic": "0.1.21",
     "@langchain/cohere": "0.0.10",
     "@langchain/community": "0.2.13",
     "@langchain/core": "0.2.9",
     "@langchain/google-genai": "0.0.16",
+    "@langchain/google-vertexai": "^0.0.19",
```

Comment:
Please pin these deps to a static version as it's more reliable with Langchain because sometimes. There are breaking changes in "minor" versions